Website/DesktopModules/Hotcakes/Core/Areas/Account/Controllers/WishListController.cs

```diff
@@ -112,7 +112,10 @@ private List<SavedItemViewModel> PrepItems(List<WishListItem> items)
             foreach (var item in items)
             {
                 var p = HccApp.CatalogServices.Products.FindWithCache(item.ProductId);
-                item.ProductShortDescription = p.Options.CartDescription(item.SelectionData.OptionSelectionList);
+                if (p.HasOptions())
```

Comment:
This will always be true if the product has options, regardless to whether any have been selected prior to adding it to the cart or wish list.  You'll need to evaluate item.SelectionData.OptionSelectionList instead.

Reply:
@ketangarala This is still an issue.

Reply:
As per your Suggestion item.SelectionData.OptionSelectionList is replaced in condition to check nullable and code is Committed , so please check last committed change.